examples/b2c/checkout/integrations/classes/B2BDeliverySample.cls

```diff
@@ -60,13 +62,13 @@ global class B2BDeliverySample implements sfdc_checkout.CartShippingCharges {
         return integStatus;
     }
 
-    private ShippingOptionsAndRatesFromExternalService[] getShippingOptionsAndRatesFromExternalService () {
+    private ShippingOptionsAndRatesFromExternalService[] getShippingOptionsAndRatesFromExternalService (String siteLanguage) {
         ShippingOptionsAndRatesFromExternalService[] shippingOptions = new List<ShippingOptionsAndRatesFromExternalService>();
         Http http = new Http();
         HttpRequest request = new HttpRequest();
         Integer SuccessfulHttpRequest = 200;
         // To access the service below, you may need to add endpoint = https://b2b-commerce-test.herokuapp.com in Setup | Security | Remote site settings.
-        request.setEndpoint('https://b2b-commerce-test.herokuapp.com/calculate-shipping-rates-winter-21');
+        request.setEndpoint('https://b2b-commerce-test.herokuapp.com/calculate-shipping-rates-winter-21-with-lang?lang=' + siteLanguage);
```

Comment:
why do we need to change the url?

Reply:
I added a new url that supports language as a query param and left the original one as is so that it is backward compatible.

Reply:
Adding a query parameter without changing url can also be backward compatible right?

Reply:
Well .. you are right but I wanted to be more specific that there is a new version that supports language.